aten/src/ATen/CMakeLists.txt

```diff
@@ -427,17 +427,25 @@ CONFIGURE_FILE(ATenConfig.cmake.in "${CMAKE_CURRENT_BINARY_DIR}/cmake-exports/AT
 INSTALL(FILES "${CMAKE_CURRENT_BINARY_DIR}/cmake-exports/ATenConfig.cmake"
   DESTINATION "${AT_INSTALL_SHARE_DIR}/cmake/ATen")
 
+if(INTERN_BUILD_MOBILE)
```

Comment:
This looks ok to me. Alternatively maybe you could append INSTALL_HEADERS list above, in those if(USE_CUDA) / if(CUDNN_FOUND) / if(USE_ROCM) sections where they expand ATen_CUDA_INCLUDE / ATen_HIP_INCLUDE and set all_cuda_cpp / all_hip_cpp? Technically whether to include these headers should be determined by those USE_XXX macros?